compiler/test/dotty/tools/backend/jvm/InlineBytecodeTests.scala

```diff
@@ -101,6 +101,399 @@ class InlineBytecodeTests extends DottyBytecodeTest {
     }
   }
 
+  @Test def i4947a = {
+    val source = """class Foo {
+                   |  inline def fact[T](inline i: Int)(f: => T): Int = {
+                   |    printStack("track", i) // line 3
+                   |    printStack("track", i) // line 4
+                   |    f
+                   |    if (i == 0)
+                   |      1 // line 7
+                   |    else {
+                   |      i * fact(i-1)(f) // line 9
+                   |    }
+                   |  }
+                   |
+                   |  def printStack(tag: String, i: Int): Unit = { // line 13
+                   |    println(s"$tag (i = $i): ${new Exception().getStackTrace().apply(1)}") // line 14
+                   |  }
+                   |
+                   |  def main(args: Array[String]): Unit = {
+                   |    fact(0) { // line 18
+                   |      fact(2) { // line 19
+                   |        printStack("main1", -1) // line 20
+                   |        printStack("main2", -1) // line 21
+                   |      }
+                   |    } // line 23
+                   |  }
+                   |}
+                 """.stripMargin
+
+    checkBCode(source) { dir =>
+      val clsIn      = dir.lookupName("Foo.class", directory = false).input
+      val clsNode    = loadClassNode(clsIn, skipDebugInfo = false)
+
+      val track = clsNode.methods.asScala.find(_.name == "fact")
+      assert(track.isEmpty, "method `fact` should have been erased")
+
+      val main = getMethod(clsNode, "main")
+      val instructions = instructionsFromMethod(main)
+      val expected =
```

Comment:
Can we do better that these huge and unmaintainable bytecode tests? See e04918e3fe688883443f35659a74e85f5347856d. I suggest having tests that only assert the presence of the instructions you are expecting (I believe `LineNumber` here)

Reply:
I will not keep this test. I added them to try to figure out the source of the bug I'm chasing.
Now I have a minimized version of it that will eventually replace it.

Reply:
I opened #6085 with the minimized version of that issue